src/transformState.js

```diff
@@ -6,6 +6,30 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (type) {
+      case 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of keysToRemove) {
+          delete state[key];
+        }
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+      default:;
```

Comment:
You can add just break here
```suggestion
      default:
        break;
```